lib/resources/http.rb

```diff
@@ -25,13 +25,14 @@ class Http < Inspec.resource(1)
     "
 
     # rubocop:disable ParameterLists
-    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil)
+    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil, ssl_verify: true)
```

Comment:
This feels a bit like YAGNI since we don't know exactly what the future looks like for this resource, but I'll defer to @chris-rock's guidance on whether to accept a hash or a boolean here.